stdlib/LinearAlgebra/test/diagonal.jl

```diff
@@ -458,6 +458,30 @@ end
     end
 end
 
+@testset "testing istriu, istril, isdiag for Diagonal" begin
+        D1 = Diagonal([[1 0; 3 4], [5 0; 7 8]])
```

Comment:
Should be 4 spaces for indentation, and no newline before the `end`.

Reply:
ok, cool